dev-haskell/diagrams-lib/diagrams-lib-1.4.3-r1.ebuild

```diff
@@ -51,22 +54,18 @@ DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-1.22.2.0
 	test? ( dev-haskell/numeric-extras
 		>=dev-haskell/quickcheck-2.7
-		>=dev-haskell/tasty-0.10
-		>=dev-haskell/tasty-hunit-0.9.2
-		>=dev-haskell/tasty-quickcheck-0.8 )
+		>=dev-haskell/tasty-0.10 <dev-haskell/tasty-1.3
```

Comment:
Does this upper bound definitely need to be added back in? The diff suggests it was previously removed. If we can leave it off that will help with bumping up tasty over time, as I think they're up to 1.4 now.

Reply:
This will be a common thread in this PR, so please refer to [this response](https://github.com/gentoo-haskell/gentoo-haskell/pull/1118#discussion_r556518075)